src/sdk.ts

```diff
@@ -0,0 +1,59 @@
+import Users from './users'
+import Things from './things'
```

Comment:
where are you using things in this pr? 
Also remove the commented out code. It will be added in subsequent pr's as needed.